baseclasses/helper/file_parser/necc_excel_parser.py

```diff
@@ -0,0 +1,145 @@
+# MIT License
+
+# Copyright (c) 2019
+
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+import pandas as pd
+
+from baseclasses.chemical_energy import NECCFeedGas, GasFEResults
+
+def read_potentiostat_data(file):
+    data = pd.read_excel(file, sheet_name='Raw Data', header=1)
+
+    datetimes = pd.to_datetime(data['time/s']).dropna()
+
+    # TODO decide with Christina which col name to use
+    if {'<I>/mA', 'Ewe/V'}.issubset(data.columns):
+        current = data['<I>/mA'].dropna()
+        working_electrode_potential = data['Ewe/V'].dropna()
+    elif {'I/mA', '<Ewe/V>'}.issubset(data.columns):
+        current = data['I/mA'].dropna()
+        working_electrode_potential = data['<Ewe/V>'].dropna()
```

Comment:
Still to be discussed with NECC group:  what are the final column names for thermocouple data in excel?

Reply:
Decided to do it all without <> :white_check_mark: